examples/seismic/viscoacoustic/viscoacoustic_example.py

```diff
@@ -35,9 +38,12 @@ def run(shape=(50, 50), spacing=(20.0, 20.0), tn=1000.0,
     return (summary.gflopss, summary.oi, summary.timings, [rec])
 
 
-def test_viscoacoustic():
+@pytest.mark.parametrize("dtype", [("float32"), ("float64")])
+def test_viscoacoustic(dtype):
+
+    dtype = eval((''.join(['np.', dtype])))
```

Comment:
I'd just put a dict in utils or something `{'float32': np.float32, 'float64': np.float64}